tesseract_ros/tesseract_msgs/msg/PlanningRequestArchive.msg

```diff
@@ -4,6 +4,8 @@ string name # The name of the planner to use
 string instructions # This should an xml string of the command language instructions
 string seed # This should an xml string of the command language instructions (Optional)
 uint16 num_threads # This is the number of thread to use during planning
+bool debug # Enable debug content
+bool profile # Enable profiling of the planning request
```

Comment:
As we add more things to the planning request, it makes me think more and more that we need a basic ProcessInput msg. Then you can run your serialized messages through the entire process manager not just the motion planner.

Reply:
At the moment, the ProcessInput would be basically the same as the PlanningRequest. It would just not have some things like debug, threads, etc.